template/rwa-onboarding/Goerli-DssSpellCollateral.sol

```diff
@@ -61,149 +71,177 @@ contract DssSpellCollateralAction {
-    address constant RWA999_A_INPUT_CONDUIT  = 0x0000000000000000000000000000000000000000; // TODO CES team should provide
-    address constant RWA999_A_OUTPUT_CONDUIT = 0x0000000000000000000000000000000000000000; // TODO CES team should provide
-    address constant RWA999_A_OPERATOR       = 0x0000000000000000000000000000000000000000; // TODO CES team should provide
-    address constant RWA999_A_MATE           = 0x0000000000000000000000000000000000000000; // TODO CES team should provide
-
-    uint256 constant RWA999_A_INITIAL_DC     = 80000000 * RAD;     // TODO RWA team should provide
-    uint256 constant RWA999_A_INITIAL_PRICE  = 52 * MILLION * WAD; // TODO RWA team should provide
-    uint48  constant RWA999_A_TAU            = 1 weeks;            // TODO RWA team should provide
-
-    /**
-     * @notice MIP13c3-SP4 Declaration of Intent & Commercial Points -
-     *   Off-Chain Asset Backed Lender to onboard Real World Assets
-     *   as Collateral for a DAI loan
-     *
-     * https://ipfs.io/ipfs/QmdmAUTU3sd9VkdfTZNQM6krc9jsKgF2pz7W1qvvfJo1xk
-     */
-    string constant DOC                      = ""; // TODO Reference to a documents which describe deal (should be uploaded to IPFS)
-
-    uint256 constant REG_CLASS_RWA           = 3;
-
-    // --- DEPLOYED COLLATERAL ADDRESSES ---
-
-    function onboardNewCollaterals() internal {
-        ChainlogAbstract CHANGELOG       = ChainlogAbstract(DssExecLib.LOG);
-        IlkRegistryAbstract REGISTRY     = IlkRegistryAbstract(DssExecLib.reg());
-        address MIP21_LIQUIDATION_ORACLE = CHANGELOG.getAddress("MIP21_LIQUIDATION_ORACLE");
-        address MCD_VAT                  = CHANGELOG.getAddress("MCD_VAT");
-        address MCD_JUG                  = CHANGELOG.getAddress("MCD_JUG");
-        address MCD_SPOT                 = CHANGELOG.getAddress("MCD_SPOT");
-
-        // RWA999-A collateral deploy
-
-        // Set ilk bytes32 variable
-        bytes32 ilk = "RWA999-A";
-        uint256 decimals = DSTokenAbstract(RWA999).decimals();
+    uint256 internal constant WAD = 10**18;
+
+    // -- RWAXXX MIP21 components --
+    address internal constant RWAXXX                         = 0x0000000000000000000000000000000000000000; // TODO
+    address internal constant MCD_JOIN_RWAXXX_A              = 0x0000000000000000000000000000000000000000; // TODO
+    address internal constant RWAXXX_A_URN                   = 0x0000000000000000000000000000000000000000; // TODO
+    address internal constant RWAXXX_A_JAR                   = 0x0000000000000000000000000000000000000000; // TODO
+    address internal constant RWAXXX_A_OUTPUT_CONDUIT        = 0x0000000000000000000000000000000000000000; // TODO
+    // Jar and URN Input Conduits
+    address internal constant RWAXXX_A_INPUT_CONDUIT_URN     = 0x0000000000000000000000000000000000000000; // TODO
+    address internal constant RWAXXX_A_INPUT_CONDUIT_JAR     = 0x0000000000000000000000000000000000000000; // TODO
```

Comment:
Maybe add a comment that `RWAXXX_A_INPUT_CONDUIT_JAR` is only required if fee payments are made in a token other than Dai.